.travis.yml

```diff
@@ -7,4 +7,3 @@ after_success:
   - npm run coverage
 script:
   - npm test
-  - npm run e2e-test
```

Comment:
so there are no more e2e test now?

Reply:
@mykola-mokhnach That's right. The only e2e test was the S3 test.